djstripe/settings.py

```diff
@@ -137,3 +108,15 @@ def get_subscriber_model():
 )
 
 DJSTRIPE_WEBHOOK_URL = getattr(settings, "DJSTRIPE_WEBHOOK_URL", r"^webhook/$")
+
+
+def get_plan_list():
```

Comment:
I don't think there's a need for this method if it's only one line calling objects.all()